ts/Core/Series/Series.ts

```diff
@@ -4778,8 +4778,14 @@ class Series {
                 'dataLabelsGroup',
                 'markerGroup',
                 'tracker',
-                'tt'
-            ] as ('group'|'dataLabelsGroup'|'markerGroup'|'tracker'|'tt')[]
+                'tt',
+                'lastPrice',
+                'lastPriceLabel',
+                'lastVisiblePrice',
+                'lastVisiblePriceLabel'
```

Comment:
Please, don't modify the source code for external modules support. Instead use `show`/`hide` events. Bind callbacks to them in the PriceIndication.ts and hide/show respecting elements.